test-integration/Microsoft.Azure.WebJobs.Extensions.OpenApi.Document.Tests/Post_ApplicationJson_Double_Tests.cs

```diff
@@ -44,15 +44,16 @@ public void Given_OpenApiDocument_Then_It_Should_Return_OperationRequestBodyCont
         }
 
         [DataTestMethod]
-        [DataRow("/post-applicationjson-double", "post", "text/plain", "double")]
-        public void Given_OpenApiDocument_Then_It_Should_Return_OperationRequestBodyContentTypeSchema(string path, string operationType, string contentType, string propertyType)
+        [DataRow("/post-applicationjson-double", "post", "text/plain", "number", "double")]
+        public void Given_OpenApiDocument_Then_It_Should_Return_OperationRequestBodyContentTypeSchema(string path, string operationType, string contentType, string propertyType, string propertyFormat)
         {
             var content = this._doc["paths"][path][operationType]["requestBody"]["content"];
 
             var value = content[contentType]["schema"];
 
             value.Should().NotBeNull();
             value.Value<string>("type").Should().Be(propertyType);
+            value.Value<string>("format").Should().Be(propertyType);
```

Comment:
Test Properties are same here, I think you should change this.

Reply:
Thank you for your support!